_posts/2015-09-29-Asking-Users-for-Help.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: Asking users for help
+permalink: /asking-users-for-help/
+tags: android, debugging
+category: Android
+author_id: mbobzien
+---
+
+##What’s this article about?
+
+We recently struggled with a very annoying issue in our android app. It extremely hard to debug because we were not able to reproduce it properly. Still we got quite a few reports about the bug from users. So we decided them for help debugging the app. In this article I will describe our experiences. Even if you develop for other platforms than android, our experiences might be usefull for you. So keep reading!
```

Comment:
Here I'd remove a few dots to make the reading easier:
> We recently struggled with a very annoying issue in our android app: it `was` extremely hard to debug because we were not able to reproduce it properly. Still we got quite a few reports about the bug from users, so we decided `to ask` them for help debugging the app. In this article I will describe our experiences. Even if you develop for other platforms than android, it might be `useful` for you. So keep reading!